frontend/src/metabase/dashboard/actions/cards.unit.spec.ts

```diff
@@ -264,27 +268,30 @@ describe("dashboard/actions/cards", () => {
       );
     });
 
-    it("should auto-wire parameters", async () => {
+    it("should not auto-wire parameters", async () => {
       const nextCardId = ORDERS_LINE_CHART_CARD.id;
-      const otherCardParameterMappings = checkNotNull(
-        PIE_CHART_DASHCARD.parameter_mappings,
-      );
-      const expectedParameterMappings = otherCardParameterMappings.map(
-        mapping => ({
-          ...mapping,
-          card_id: nextCardId,
-        }),
-      );
 
       const { nextDashCard } = await runReplaceCardAction({
         dashcardId: TABLE_DASHCARD.id,
         nextCardId: nextCardId,
         dashcards: [...DASHCARDS, PIE_CHART_DASHCARD],
       });
 
-      expect(nextDashCard.parameter_mappings).toEqual(
-        expectedParameterMappings,
-      );
+      expect(nextDashCard.parameter_mappings).toEqual([]);
+    });
+  });
+
+  describe("addCardToDashboard", () => {
+    it("should not auto-wire parameters", async () => {
+      const { nextDashCard } = await runAddCardToDashboard({
+        dashId: DASHBOARD.id,
+        cardId: ORDERS_LINE_CHART_CARD.id,
+        tabId: null,
+        // for auto-wiring
+        dashcards: [...DASHCARDS, PIE_CHART_DASHCARD],
+      });
+
+      expect(nextDashCard.parameter_mappings).toEqual([]);
```

Comment:
<img width="1509" alt="Screenshot 2024-06-06 at 16 45 02" src="https://github.com/metabase/metabase/assets/125459446/3ea3d7b5-efb5-4bab-a611-4fc903a3463c">

it failed on master and passed on this branch